New folder/content/js/index.js

```diff
@@ -0,0 +1,48 @@
+const btnAddInput = document.getElementById('btnAdd');
+const btnDisplay = document.getElementById('btnDisplay');
+const elementsInput = document.querySelector('.elements');
+
+var elementList = [];
+
+if(localStorage.getItem("information") !== null){
+    elementList = JSON.parse(localStorage.getItem("information"));
+    displayElements();
+}
+
+btnAddInput.addEventListener('click' , function(){
+    addElement();
+})
+
+btnDisplay.addEventListener('click' , function(){
+    displayElements();
+})
+
+
+function addElement(){
+
+    var elements = {
+        Input: elementsInput.value, 
+    };
+
+    elementList.push(elements);
```

Comment:
you can directly push elementsInput.value no need for creating another variable 
did you do that for a purpose?